src/OperatorStateRetriever.sol

```diff
@@ -159,4 +159,23 @@ contract OperatorStateRetriever {
 
         return checkSignaturesIndices;
     }
+
+    /**
+     * @notice this function returns the quorumBitmaps for each of the operators in the operatorIds array at the given blocknumber
+     * @param registryCoordinator is the AVS registry coordinator to fetch the operator information from
+     * @param operatorIds are the ids of the operators to get the quorumBitmaps for
+     * @param blockNumber is the block number to get the quorumBitmaps for
+     */
+    function getQuorumBitmapsAtBlockNumber(
```

Comment:
I wonder what's the failure cases for this function, like:
- for an operatorID, in what cases it'll fail to get bitmap? Do we distinguish between not found v.s. some real error?
- does this function require all operatorIDs to find bitmap to succeed ("ALL-OF" v.s. "ONE-OF")? If it's ONE-OF semantics, what will be included in quorumBitmaps[i] if operatorIds[i] failed to get result?

Reply:
these are excellent questions to be asking
AFAICT if it doesn't find a value for a single operator, it should revert entirely.